lib/src/binary_codec.dart

```diff
@@ -292,6 +411,26 @@ class PostgresBinaryDecoder extends Converter<Uint8List, dynamic> {
     }
   }
 
+  List<T> readListBytes<T>(
+      ByteData buffer, T Function(int offset, int length) valueDecoder) {
+    final decoded = [].cast<T>();
+
+    try {
+      final size = buffer.getInt32(12);
+
+      var offset = 20;
+      for (var i = 0; i < size; i++) {
+        final len = buffer.getInt32(offset);
+        decoded.add(valueDecoder(offset + 4, len));
+        offset += 4 + len;
+      }
+
+      return decoded;
+    } on RangeError catch (_) {
+      return decoded;
```

Comment:
This feels strange: why would we have RangeError?

Reply:
Done